en/docs/references/concepts/claim-management.md

```diff
@@ -4,59 +4,57 @@ A claim is a piece of information about a particular subject. It can be
 anything that the subject is owned by or associated with, such as name,
 group, preferences, etc. A claim provides a single and general notion to
 define the identity information related to the subject. Claim-based
-identity is a common way for any application to acquire the identity
+identity is a common way for any application to acquire identity
 information. It provides a consistent approach for all applications by
 hiding the lower level implementation. Claims are also used in identity
-propagation which is the replication of authenticated **identities**,
+propagation which is the replication of authenticated identities,
 by packaging the claims into one or more tokens (such as SAML). These
-tokens are then issued by an issuer (eg., [security token
-service](TO-DO:../../get-started/single-sign-on-and-identity-federation#ws-trust)
+tokens are then issued by an issuer (eg., [security token service](../authentication/intro-ws-trust)).
 
 A user claim is a claim that is related to a user. It can be used to
 specify information that is directly related to the user, such as claims
 related to the street address, username, email, first name, and more.
 
 An identity claim is a special claim related to identity management. It
 can be used to specify information about the user account or the state
-of a user account such as the lastLoginTime, accountDisabled and
+of a user account such as the lastLoginTime, accountDisabled, and
 accountLocked claims. Identity claims are identified by the claim URI.
 All identity claims have the term "identity" appended to the claim URI
 as follows:
-`http://wso2.org/claims/identity/accountDisabled. `
+`http://wso2.org/claims/identity/accountDisabled`
 
 The Claim Management component of [WSO2
```

Comment:
I don't think we maintain the carbon docs anymore. Better to not give this link.